median/main.go

```diff
@@ -0,0 +1,35 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"sort"
+)
+
+func median(intsArr []int) float64 {
+	var answer float64
+	sort.Ints(intsArr)
+	log.Println("Sorted list:", intsArr)
+	val := len(intsArr) / 2
+	log.Println("Half value: ", val)
+	if len(intsArr)%2 == 1 {
+		log.Println("%2 == 1")
+		answer = float64(intsArr[val])
+		log.Println("Answer:", answer)
```

Comment:
You're already printing the answer in the main function, so you don't really need to do it here as well, and then the variable `answer` is redundant and you can just return the value.